scripts/ipxe-dts.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+SCRIPTDIR=$(readlink -f "$(dirname "$0")")
+
+error_exit() {
+    _error_msg="$1"
+    echo "$_error_msg"
+    exit 1
+}
+
+error_check() {
+    _error_code=$?
+    _error_msg="$1"
+    [ "$_error_code" -ne 0 ] && error_exit "$_error_msg : ($_error_code)"
+}
+
+userIP=$(ip route get 1 | sed 's/^.*src \([^ ]*\).*$/\1/;q')
+error_check "cannot get user IP"
+
+distroVersion=$(cat "$SCRIPTDIR"/../meta-dts-distro/conf/distro/dts-distro.conf | grep -F "DISTRO_VERSION" | awk -F'"' '{print $2}')
```

Comment:
We can use quotations on whole path so `"$SCRIPTDIR/../meta-dts-distro/conf/distro/dts-distro.conf"`

Reply:
@TomaszAIR done [here](https://github.com/Dasharo/meta-dts/pull/96/files#diff-50ccba31fdf15d04ffc9cadbcbbfc4ceebd19e4076523602a1ee758e9a72cc68R20)